openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -617,7 +623,20 @@ Cache-Control: no-store
     "token_type": "bearer",
     "expires_in": 86400,
     "c_nonce": "tZignsnFbp",
-    "c_nonce_expires_in": 86400
+    "c_nonce_expires_in": 86400,
+    "authorization_details": [
+      {
+        "type": "openid_credential",
+        "format": "jwt_vc_json",
+        "credential_definition": {
+          "type": [
+              "VerifiableCredential",
+              "UniversityDegreeCredential"
+          ]
+        },
+        "c_instance_identifiers": [ "CivilEngineeringDegree-2023", "ElectricalEngineeringDegree-2023" ]
+      }
+    ]
```

Comment:
```suggestion
    "c_nonce_expires_in": 86400,
    "authorization_details": [
      {
        "c_instance_identifiers": [ "CivilEngineeringDegree-2023", "ElectricalEngineeringDegree-2023" ]
      }
    ]
```

Reply:
Are the details really relevant? I would see benefit in keeping the Token Response format agnostic

Reply:
If we get multiple credentials offered in a Credential Offer, we haven't actually picked one, so therefore we need those details? We could even use `credentials_supported_identifier` from #86 to solve this easier then.

Reply:
On your second comment, nothing prevents AS/RS from using the same value for `credentials_upported_identifier` and `c_instance_identifier`, but conceptually, I really think these parameters needs to be separate.

> Are the details really relevant?

I honestly don't know.. RAR spec says "The AS MAY omit values in the authorization_details to the client." in https://datatracker.ietf.org/doc/html/rfc9396#name-token-response. so I think that means that structure below is technically valid, because "type" is the only required parameter if I am correct. but I am not sure if format and type should be omitted here, maybe when there is only one credential being issued, but not when there are multiple, because having type and format here provides a clear binding between an identifier and type/format.

```suggestion
    "c_nonce_expires_in": 86400,
    "authorization_details": [
      {
        "type": "openid_credential",
        "c_instance_identifiers": [ "CivilEngineeringDegree-2023", "ElectricalEngineeringDegree-2023" ]
      }
    ]
```

Reply:
You seem to want to make most of the requests in VCI format agnostic. why is that..?